en/releases/centreon-commercial-extensions.md

```diff
@@ -18,6 +18,10 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP release notes
 
+### 20.04.3
+
+* Filter on the full name of the maps and geoviews on the homepage
+
```

Comment:
Fix on media access from centreon map server

Reply:
Ok I push it